static/src/javascripts/projects/common/modules/navigation/new-header.js

```diff
@@ -362,11 +348,59 @@ const trackRecentSearch = (): void => {
 
 const saveSearchTerm = (term: string) => local.set(SEARCH_STORAGE_KEY, term);
 
+const showMoreButton = (): void => {
+    fastdom
+        .read(() => {
+            const moreButton = document.querySelector('.js-show-more-button');
+            const subnav = document.querySelector('.js-expand-subnav');
+            const subnavList = document.querySelector(
+                '.js-get-last-child-subnav'
+            );
+
+            if (subnav && subnavList) {
```

Comment:
Might be an overly cautious thing, but if `subnav` or `subnavList` are falsey, this callback implicitly returns `undefined` and the destructuring on line 371 will throw an error. It might be best to return `Promise.reject()` at this point, and add a catch to the bottom of this promise block that does nothing.

Reply:
I tried that, but I don't know if I can use that on a fastdom promise. Or at least, I don't understand it well enough to do that 😬 .

I am going on holiday now, but thank you so much for reviewing!

Reply:
Your solution works too!